iseq.c

```diff
@@ -2733,11 +2733,11 @@ rb_iseq_disasm_recursive(const rb_iseq_t *iseq, VALUE indent)
             }
 
             snprintf(argi, sizeof(argi), "%s%s%s%s%s%s",	/* arg, opts, rest, post, kwrest, block */
-                     body->param.lead_num > li ? "Arg" : "",
+                     (body->param.lead_num > li) ? (i == (body->local_table_size - 1) && body->param.flags.ambiguous_param0 ? "AmbiguousArg" : "Arg") : "",
```

Comment:
Unclear what `(i == (body->local_table_size - 1)` is checking.

Reply:
Yeah, it was meant to double-check that we are only looking at the first parameter, but I can actually just remove that.